app/classes/lookup/names.rb

```diff
@@ -60,6 +72,20 @@ def original_matches
     end.flatten.uniq.compact
   end
 
+  def numeric_list?(list)
+    list.all? do |item|
+      case item
+      when Numeric
+        true
+      when String
+        # Handles integers, floats, scientific notation, and negative numbers
+        item.match?(/\A-?\d*\.?\d+(?:[eE][-+]?\d+)?\z/)
```

Comment:
```suggestion
item.match?(/^-?\d+$/)
```
Since this is supposed to be a list of IDs, i do not think it should match "floats, scientific notation, and negative numbers", right?

Reply:
Thanks!  Updated.